### ISSUE TITLE
bring back stench of death in some way

### ISSUE THREAD

**Issue by lxative** [niceh26/Symbiosis#3]
would appreciate it if stench of death was added back, or made into its own mod.

**Comment by MadKattz** [comment]
Agreed, not sure why it was removed as I'm pretty sure there was already a way to disable it in the config
